specs/address_book_list/01_main.rst

```diff
@@ -0,0 +1,140 @@
++--------+------------------+
+| issue  | title            |
++--------+------------------+
+| 81_    | Address Book List|
++--------+------------------+
+
+.. sectnum::
+
+.. _81: https://github.com/gnosis/safe/issues/81
+
+
+Address Book List
+=================
+
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Problem Definition
+-------------------------------
+
+- User can recognize names better than full addresses
+- User wants to associate addresses to names
+- User can see the address book entries and their details
+- User can search the address book entries
+- The address book is not synced (ie.: it is part of the local storage only)
+
+Inputs
+-----------
+
+- Address book entries stored locally (local database)
+- Search with name or Ethereum address
+
+Outputs
+------------
+
+- Formatted list to be displayed to the user. Each entry has:
+    - Identicon
+    - Name
+    - Ethereum address
+
```

Comment:
We display and store only checksummed addresses?

If the user provides a non-checksummed address, should we checksum it?

Reply:
Didn't we have a doc were we outlined how to display addresses (Checksummed: first 4 and last 4)

Reply:
> We display and store only checksummed addresses?
> 
> If the user provides a non-checksummed address, should we checksum it?

We store the numeric value of the address (so checksum doesn't matter here). We can specify here if we only accept checksummed addresses. Right now it doesn't need to be checksummed.

We can create a different spec just for displaying addresses across the app what do you think? And we can reference that spec whenever an address is shown.